test/clearingHouse/ClearingHouse.newMarket.ts

```diff
@@ -238,27 +243,28 @@ describe("ClearingHouse new market listing", () => {
             await addOrder(fixture, bob, 1, 10, lowerTick, upperTick, false, baseToken3.address)
 
             // open market
-            await exchange.setMaxTickCrossedWithinBlock(baseToken.address, "1000")
-            await exchange.setMaxTickCrossedWithinBlock(baseToken3.address, "1000")
+            await exchange.setMaxTickCrossedWithinBlock(baseToken.address, "100000")
+            await exchange.setMaxTickCrossedWithinBlock(baseToken3.address, "100000")
 
             // bob open position in baseToken, baseToken3 market
             await q2bExactInput(fixture, bob, 800, baseToken.address)
             await q2bExactInput(fixture, bob, 800, baseToken3.address)
 
-            // pause baseToken3 market
+            // pause any swap in baseToken3 market, not delist
             await exchange.setMaxTickCrossedWithinBlock(baseToken3.address, "0")
 
-            // drop baseToken market price
-            mockedBaseAggregator.smocked.latestRoundData.will.return.with(async () => {
-                return [0, parseUnits("0.000001", 6), 0, 0, 0]
-            })
-
-            // drop baseToken3 market price
-            mockedBaseAggregator3.smocked.latestRoundData.will.return.with(async () => {
-                return [0, parseUnits("0.000001", 6), 0, 0, 0]
-            })
-
-            await mintAndDeposit(fixture, davis, 10000)
+            // drop baseToken mark price
+            await b2qExactOutput(fixture, alice, 10000, baseToken.address)
+            // await b2qExactOutput(fixture, alice, 10000, baseToken3.address)
+            await forwardBothTimestamps(clearingHouse, 1800)
+            await closePosition(fixture, alice, 0, baseToken.address)
```

Comment:
Is this something we'd like to test? 
If yes, should be in a standalone case instead of here in `beforeEach`.

Reply:
They are pre-work for this test case. I refine the test case and add more comments for describing the test scenario.
Fixed in e0f729d8ac3d253d6fd6c9dae864242370691a13